SoftwareGuide/Latex/Appendices/CodingStyleGuide.tex

```diff
@@ -2464,13 +2478,16 @@ \subsection{Alignment}
 
 \small
 \begin{minted}[baselinestretch=1,fontsize=\footnotesize,linenos=false,bgcolor=ltgray]{cpp}
-InputPixelType  m_ForegroundValue;
-OutputPixelType m_BackgroundValue;
+bool                        m_Quiet{ false };
+bool                        m_TestAbort{ false };
+std::string                 m_Comment;
+itk::ProcessObject::Pointer m_Process;
 
-unsigned int m_MaximumIterations;
+CommandType::Pointer m_StartFilterCommand;
```

Comment:
Type names of same length naturally result in aligned variables, so this choice does not demonstrate the point well.

Reply:
> Type names of same length naturally result in aligned variables

Correct. Sorry again for the poor choice. Not easy to find the right examples in the code base; its consistency concerning the initialization and Doxygen comments has room for improvement.

Resolving since I believe 55ea845 now demonstrates the point well.